azurelinuxagent/common/protocol/wire.py

```diff
@@ -77,6 +77,13 @@ def __init__(self, endpoint):
     def detect(self, init_goal_state=True, save_to_history=False):
         self.client.check_wire_protocol_version()
 
+        # Send telemetry if protocol endpoint is not the known WireServer endpoint.
+        endpoint = self.get_endpoint()
+        if endpoint is not None and endpoint != KNOWN_WIRESERVER_IP:
+            message = 'Protocol endpoint is not known wireserver ip: {0}'.format(endpoint)
+            logger.info(message)
+            add_event(op=WALAEventOperation.ProtocolEndpoint, message=message)
```

Comment:
This event didn't seem to fit in any existing operation so I added ProtocolEndpoint